running-coreos/cloud-providers/interoute/index.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: docs
+title: Interoute VDC
+category: running_coreos
+sub_category: cloud_provider
+weight: 11
+---
+
+# Running CoreOS on Interoute VDC
+
+Interoute Communications Limited is the owner operator of Europe's largest cloud services platform, which encompasses over 67,000 km of lit fiber, 12 data centres, 13 Virtual Data Centres and 31 colocation centres, with connections to 195 additional third-party data centres across Europe. 
+
+To run a single CoreOS node on Interoute VDC the following is assumed:
+
+* You have an Interoute VDC account. You can easily [sign up for a free trial](http://cloudstore.interoute.com/main/TryInterouteVDCFREE) (no credit card required).
+* You have the  Cloudmonkey command line tool installed and configured on the computer that you are working on. Instructions on how to install and configure Cloudmonkey so that it can communicate with the VDC API can be found in the [Introduction to the VDC API](http://cloudstore.interoute.com/main/knowledge-centre/library/vdc-api-introduction-api).
+* You have installed OpenSSH client software. This is usually already installed in Linux and Mac OS. For Windows it can be downloaded at the [OpenSSH website](http://www.openssh.com/).
+
+Note: In the following steps, commands beginning with '$' are to be typed into the command line, and commands beginning '>' are to be typed into Cloudmonkey. 
+
+## Cloudmonkey Setup
+
+First you should open a new terminal or command prompt window and start Cloudmonkey by typing:
+
+```cloudmonkey
+$ cloudmonkey set display table && cloudmonkey sync && cloudmonkey
+252 APIs discovered and cached
+Apache CloudStack cloudmonkey 5.3.0. Type help or ? to list commands.
+
+Using management server profile: local 
+
+(local) >
+```
+After running this, you should see that Cloudmonkey has started successfully and that it's ready to accept API calls.All of the VDC API commands that can be accepted by Cloudmonkey can be found in the [API Command Reference](http://cloudstore.interoute.com/main/knowledge-centre/library/api-command-reference).
+
+## Deploy a CoreOS node
+
+The following API call from Cloudmonkey is used to deploy a new virtual machine running CoreOS in VDC:
+
+```cloudmonkey
+> deployVirtualMachine serviceofferingid=85228261-fc66-4092-8e54-917d1702979d zoneid=f6b0d029-8e53-413b-99f3-e0a2a543ee1d templateid=73bc5066-b536-4325-8e27-ec873cea6ce7 networkids=e9e1220b-76c8-47cd-a6c2-885ffee49972 keypair=CoreOS-Key01 name=DockerTutorialVM01
```

Comment:
It would be much easier to read this if you used this format:

```
deployVirtualMachine serviceofferingid=85228261-fc66-4092-8e54-917d1702979d \
  zoneid=f6b0d029-8e53-413b-99f3-e0a2a543ee1d \
  templateid=73bc5066-b536-4325-8e27-ec873cea6ce7 \
  networkids=e9e1220b-76c8-47cd-a6c2-885ffee49972 \
  keypair=CoreOS-Key01 \
  name=DockerTutorialVM01
```

Potential customers of yours might not know what a `serviceofferingid` or `templateid` are, or where to find them. I'll leave it up to you to determine if that needs more explanation.